packages/data-point/lib/accumulator/factory.js

```diff
@@ -8,11 +8,18 @@ const LocalsFactory = require('../locals/factory')
  */
 function Accumulator () {
   this.value = undefined
+  this.isResolved = false
   this.locals = undefined
   this.values = undefined
   this.reducer = undefined
   this.trace = false
   this.context = undefined
+  this.resolveTransformWith = function (value) {
+    return {
```

Comment:
this value needs to be more specific isResolved could be used by anyone else, this is one of the issues with this solution. Maybe using Symbol? not sure yet